app/templates/errors/403.html

```diff
@@ -0,0 +1,14 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Access denied - 403 – Digital Marketplace{% endblock %}
+
+{% block content %}
+  <div class="page-container">
+    {% with heading = "You don’t have permission to view this page" %}
```

Comment:
This might be better as "you don't have permission to perform this action" - as the restricted users will hit this most often after trying to press buttons they shouldn't, rather than trying to view pages.